biocrnpyler/sbmlutil.py

```diff
@@ -225,18 +223,28 @@ def _create_reactants(reactant_list, sbml_reaction, model):
         # What to do when there are multiple species with same name?
```

Comment:
Do we want to remove this comment?

Reply:
Did we do something about this? Is it printed out as an error/warning some place?

Reply:
I think we should definitely have a warning